src/style.css

```diff
@@ -1,3 +1,71 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
The font-family declaration includes 'Roboto' but does not provide a fallback for sans-serif fonts. Consider adding 'sans-serif' as a fallback to ensure consistent font rendering across different browsers and systems.